internal/util/relations.go

```diff
@@ -35,10 +37,31 @@ func (relations *ForeignRelations) GetCombinations() []Rel {
 				cartesianProduct = append(cartesianProduct, Rel{Consumer: consumer})
 			}
 		}
-	} else {
-		for _, consumerGroup := range relations.ConsumerGroup {
-			cartesianProduct = append(cartesianProduct, Rel{ConsumerGroup: consumerGroup})
+	} else if len(relations.ConsumerGroup) > 0 {
+		groups := relations.ConsumerGroup
+		if len(relations.Route)+len(relations.Service) > 0 {
```

Comment:
Could we use `len(relations.Route) > 0 || len(relations.Service) > 0` to let this be more intuitive?